packages/cdk8s-cli/lib/import/jsii.ts

```diff
@@ -48,6 +48,13 @@ export async function jsiiCompile(workdir: string, options: JsiiCompileOptions)
     jsii: {
       outdir: "dist",
       targets: {
+        java: {
+          package: main,
+          maven: {
+            groupId: main,
+            artifactId: main
+          }
+        },
```

Comment:
JSII config. Naming seemed reasonable. Produces:

```xml
  <groupId>k8s</groupId>
  <artifactId>k8s</artifactId>
  <version>0.0.0</version>
  <packaging>jar</packaging>
```

Reply:
groupId and artifactId is fine. I am wondering about the java package. Feels odd that this will just go to "k8s". I think it should be `cdk8s.imports.k8s`, no?

Reply:
That seems reasonable to me! I'll switch it up.